src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -100,11 +100,11 @@ export const TransactionDetailsSummaryItem: FC<TransactionDetailsSummaryItemProp
       if (selectedShippingQuote) {
         return `${
           shippingQuoteDisplayNames[selectedShippingQuote.typeName]
-        } delivery`
+        } delivery*`
       }
     }
 
-    return "Shipping"
+    return "Shipping*"
```

Comment:
Same comment as above, I dont think we want to show the `*` for a footnote on buy now orders correct? Buy nows are non-negotiable and shouldn't be expected to change